README.md

```diff
@@ -8,6 +8,58 @@ How to use:
 * Start server: go run main.go server
 * Client call to check health status of server: go run main.go client.
 
+### Instruction for Windows Service Installation
+
+If you want to install harness-docker-runner as a service in windows please follow the bellow instructions.
+
+#### Instruction from creating msi from Exe file.
+* Here is official reference document: https://github.com/mh-cbon/go-msi
+* install go-msi package
+* create the binary for harness-docker-runner-svc
+```
+  go build -o harness-docker-runner.exe
+```
+* run the below command from root directory to create msi
+```
+  go-msi make --msi harness-docker-runner-svc.msi --version your_version
+```
+
+#### Installation
+* Download the msi (harness-docker-runner-svc.msi) from latest releases
+* Double click the msi to start the installation process (If you see any error while opening the file please follow the instruction from below url)
+* https://answers.microsoft.com/en-us/windows/forum/all/this-installation-package-could-not-be/d6d913e9-aac7-429a-ac0d-c39ad3a7c5eb
+* Accept the liecence, click next and finish the installation.
+* You can test your service availibility by running below command in your cmd terminal
+```
+  curl http://localhost:3000/healthz
+```
+#### Uninstallation
+* Search and open add or remove programs from search menu
+* search for Harness-Docker-Runner
+* click Uninstall and finish.
+
+#### Error and Resolutions
+* For the below both the error, we need to give a full permission to our msi file to get it working. Since this is an extenal msi which is not tructed by windows security, it blocks it initially which can be handle by manually assigning the required permissions from Properties => Security tab.
+* If you are facing Error code 2502 or 2503, please follow the below instruction or link: https://help.krisp.ai/hc/en-us/articles/8083286001820-Error-during-installation-2502-and-2503#h_01HNCW0XCN8AJCWK84K8MQVY7Y
+* For error "This installation package cannot be opened" then please follow this link: https://answers.microsoft.com/en-us/windows/forum/all/this-installation-package-could-not-be/d6d913e9-aac7-429a-ac0d-c39ad3a7c5eb
+#### Additional Instructions
+* Service will automatically started even if you re-started your VM
+* Please do not delete the service manually from the services it may cause issue in uninstallation.
+* In case you have manually deleted the service follow below step to start service again:
+* Download the binary from release for windows
+* open the cmd as administrator and run below command
+```
+=== start the service ===
+sc.exe create Harness-Docker-Runner binPath="Absolute Path/to/your/binary server" start=auto DisplayName="Harness-Docker-Runner"
```

Comment:
make sure harness-docker-runner-svc doesn't exist before we do this.
binPath="absolute_path/to/your/harness-docker-runner.exe server"